lib/apparmortest.pm

```diff
@@ -632,6 +632,38 @@ sub adminer_database_delete {
     clear_console;
 }
 
+# Yast2 Apparmor set up
+sub yast2_apparmor_setup {
+    # Check Apparmor is active
+    systemctl("is-active apparmor");
+
+    # Turn to x11 and start "xterm"
+    select_console("x11");
```

Comment:
I think all are using select_console 'x11'; in the file.

Reply:
No